homeassistant/components/lifx_cloud/scene.py

```diff
@@ -91,7 +104,9 @@ async def async_activate(self, **kwargs: Any) -> None:
         try:
             httpsession = async_get_clientsession(self.hass)
             async with async_timeout.timeout(self._timeout):
-                await httpsession.put(url, headers=self._headers)
+                await httpsession.put(
+                    url, headers=self._headers, json=map_to_lifx_request(kwargs)
+                )
```

Comment:
IMHO, the communication logic for the integration should be extracted into a separate library first (hosted on pypi).

Reply:
I was trying to keep my change small to fix this bug, but I can pull in a library to do this. It'll replace most of the code in this integration 😂

Reply:
I looked for a suitable pypi package, and didn't find an existing one that uses async io. Do I need to create one or are these changes okay?